.wrgl/config.yaml

```diff
@@ -1078,6 +1090,10 @@ branch:
     file: data/fuse/com_west_monroe_pd.csv
     primaryKey:
       - allegation_uid
+  com-erath-pd:
+    remote: com-erath-pd
+    merge: refs/heads/main
+    file: data/fuse/com_erath_pd.csv
```

Comment:
Do these new tables not have primary key?

Reply:
My bad. Improper merge. Fixing now.